### PR TITLE
MCKIN-24630 Translations Correction for three languages

### PR DESCRIPTION
**Pull request by musmanmalik**
Translations Correction for three languages

### CODE REVIEW

**Comment by musmanmalik** [comment]
@xitij2000 Please review this PR.

**Comment by xitij2000** [comment]
@musmanmalik There are some quality errors in the CI. Could you have a look and fix the issue? It might need adding an exception to the style rules.

**Comment by musmanmalik** [comment]
@xitij2000 I have looked that. It seems no change of this PR is causing that issues.

**Comment by xitij2000** [comment]
@musmanmalik I understand that this PR did not cause the issue. However as quality requirements change each PR should clean up code to make the tests pass.

**Comment by musmanmalik** [comment]
@xitij2000 I have fixed for python 3. Now python 2 quality compromising.

**Comment by xitij2000** [comment]
> @xitij2000 I have fixed for python 3. Now python 2 quality compromising.

As I mentioned in my comment, you will need to add a pylint exception here.

File: pylintrc [hunk 1]
```diff
@@ -27,3 +27,6 @@ disable=
 
 [SIMILARITIES]
 min-similarity-lines=8
+
+[TYPECHECK]
+disable=bad-option-value
```
Comment (xitij2000) [inline pylintrc:32]:
```suggestion

```

**Review by xitij2000 — APPROVED** [review]
I've tested and this and you can simply add `super-with-arguments` to `pylintrc` and it will avoid the error.
Since we want to globally disable this check till the block is python3-only, this is the better approach. 

Once that change is made, this is good to merge. 

:+1:
- [x] I tested this: tested that it's functioning on master devstack. 
- [x] I read through the code

**Comment by musmanmalik** [comment]
@xitij2000 Please merge and create release tag.

**Comment by xitij2000** [comment]
@musmanmalik Done.

**Pull request merged**